src/app/helptext/network/interfaces/interfaces-list.ts

```diff
@@ -0,0 +1,16 @@
+import { T } from '../../../translate-marker';
+import { Validators } from '@angular/forms';
+import { rangeValidator } from '../../../pages/common/entity/entity-form/validators/range-validation';
+
+export default {
+pending_changes_text : T('You currently have pending network changes, do you wish to commit now?\
```

Comment:
As above, avoid "you":
'There are uncommitted network changes. Commit them now?\\'